highfidelity/hifi
# Add log window for the EntityScriptServer in interface.

- Added log streaming capability from the EntityScriptServer to allowed clients.
- Added log window for that purpose that subs/unsubs from the EntityScriptServer log automatically.
- Added HMD friendly log window.

## Test Plan:
- Go to a domain where you have rez rights
- `Developer > Entity Script Server Log` should be enabled
- Open it and run a simple entity server script
- Verify you can see the log
- Change domain settings so you don't have rez rights
- Log window should warn you that you have been unsubscribed from the log and the menu item should be grayed out.
- Come back to your domain.
- Restart the domain
- You should be warned the connection is lost. It should come back by itself.
- Run the essDebugWindow.js script
- Make sure the script window shows the same information and subs/unsub in the same way
